### DISCUSSION TITLE
Huge log file

### DISCUSSION THREAD

**Discussion by www516717402** [apache/airflow#35834] [Q&A]
Webserver will stop to watch log When my log file is large. How to set number of log display rows?

**Comment by potiuk** [comment]
Make sure your log is not large. There is no other option if your log is huge. Alternatively - there is an open work on makimg webserver work with huge log files https://github.com/apache/airflow/issues/31105 and if you would lilke to help by submitting a PR that can work with huge logs, you are most welcome to join > 2700 contributors of Airflow.
